apps/services/sessions/src/app/sessions/sessions.service.ts

```diff
@@ -11,11 +17,65 @@ export class SessionsService {
     private readonly sessionModel: typeof Session,
   ) {}
 
-  async findAll(user: User): Promise<Session[]> {
-    return []
+  async findAll(
+    user: User,
+    otherUser: string,
+    query: SessionsQueryDto,
+  ): Promise<SessionsResultDto> {
+    if (user.actor && !kennitala.isCompany(user.nationalId)) {
+      throw new ForbiddenException(
+        'Personal delegations are not allowed to get session data.',
+      )
+    }
```

Comment:
This is unnecessary as only authenticated user gets the `internal` scope, and procuring holders get the `internalProcuring` scope.

Reply:
Removed.